src/integrationTest/java/uk/gov/hmcts/reform/translate/BaseTest.java

```diff
@@ -1,17 +1,33 @@
 package uk.gov.hmcts.reform.translate;
 
+import org.mockito.Mock;
 import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.security.core.Authentication;
 import org.springframework.test.context.ActiveProfiles;
+import uk.gov.hmcts.reform.translate.config.TestIdamConfiguration;
+import uk.gov.hmcts.reform.translate.security.SecurityUtils;
 
-@SpringBootTest(classes = Application.class)
+import javax.inject.Inject;
+
+@SpringBootTest(classes = {
+    Application.class,
+    TestIdamConfiguration.class
+})
 @AutoConfigureMockMvc(addFilters = false)
 @ActiveProfiles("itest")
 @SuppressWarnings("HideUtilityClassConstructor")
 public class BaseTest {
 
+    @Inject
```

Comment:
Possible to use `@Autowired` here, to avoid needing to add the `javax.inject` dependency for this one single use?

Reply:
👍

Reply:
Fixed in https://github.com/hmcts/ts-translation-service/pull/10/commits/f1be3f52ad0a0ad0bb394f97bfe53b7b9e871815